Removed affiliate banners, Google+ from alternate template sets

vicos59 opened the pull request:
fixes #139 

All template sets should now be free of Affiliate banners and Google+

I left the old images in place because many are still used in Changelogs. I believe there will be a future issue to clean up Changelogs and #122 to remove obsolete images from the codebase. These are all low priority.

Note: You will have to tell Github to ignore whitespace changes or you will just see a sea of red and green. On the web page, check Hide Whitespace under the GEAR icon.

jonyo commented on src/geo_templates/default/main_page/extra_pages/advertise_with_us.tpl:
```diff
@@ -1,39 +1,29 @@
+			</div>
+			<div>
+        		<a href="https://geodesicsolutions.org">
+		        	<img style="border: medium none ;" src="{external file='images/banners/sample_468x60.jpg'}" alt="Sammple Ad Banner" title="Sammple Ad Banner">
```
Did you mean to also copy in that border style from the first one?  That goes for all of the locations, the style got copied into all of them.

vicos59 replied:
I'll fix it now. Standby.

vicos59 commented:
I've made a mess. I renamed the branch on my local git. So then I renamed it on github and now my ne commits are going to the old name. 

I am just going to close the PR and do a new one and hope it picks up the correct branch.
